reviews/serializers.py

```diff
@@ -41,6 +41,32 @@ def __repr__(self):
     return IfCanViewLogCreatorField
 
 
+class ReviewableCountingRelationshipField(RelationshipField):
+
+    def __init__(self, *args, **kwargs):
+        kwargs['related_meta'] = kwargs.get('related_meta') or {}
+        kwargs['related_meta']['include_status_counts'] = True
+        super(ReviewableCountingRelationshipField, self).__init__(*args, **kwargs)
+
+    def get_meta_information(self, metadata, provider):
+        metadata = metadata or {}
+
+        # Make counts opt in
+        show_counts = utils.is_truthy(self.context['request'].query_params.get('related_counts', False))
+        # Only include counts on detail routes
+        is_detail = not isinstance(self.context['view'], generics.ListAPIView)
+        # Weird hack to avoid being called twice
+        is_related_meta = metadata.get('include_status_counts')
```

Comment:
What causes it to be called twice? Shouldn't `include_status_counts` always be `True` here?

Reply:
One would think so, but [one](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/base/serializers.py#L716) would be [wrong](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/base/serializers.py#L718).
`include_status_counts` only exists when called with `related_metadata`

Reply:
Oh, so it is still called twice but the counts are only added when called with `related_metadata`?

Reply:
Bingo, I'll add more detail to the comment